pkg/kn/commands/source/binding/describe.go

```diff
@@ -75,11 +95,12 @@ func NewBindingDescribeCommand(p *commands.KnParams) *cobra.Command {
 			return nil
 		},
 	}
-	flags := cmd.Flags()
+	flags := command.Flags()
 	commands.AddNamespaceFlags(flags, false)
 	flags.BoolP("verbose", "v", false, "More output.")
-
-	return cmd
+	machineReadablePrintFlags.AddFlags(command)
+	command.Flag("output").Usage = fmt.Sprintf("Output format. One of: %s.", strings.Join(machineReadablePrintFlags.AllowedFormats(), "|"))
```

Comment:
this line is only needed when we're adding any other custom output format (like `-ourl`) to be able to append that custom format to help message, otherwise this line can be omitted and the default help message would be same as in this PR
@arghya88 also applies to all the recent `-o` PRs landed without `-ourl`